### PR TITLE
bug(refs DPLAN-12404): Check for location contacts in institutions

### PR DESCRIPTION
**Pull request by gruenbergerdemos**
### Ticket
DPLAN-12404

Add check for location Contacts so we don't run into an error.  

### How to review/test
Verfahren -> Institutionen verwalten -> Institution hinzufügen -> institutions list should load

### CODE REVIEW

File: client/js/components/procedure/admin/DpAddOrganisationList.vue [hunk 1]
```diff
@@ -31,22 +31,22 @@
               {{ Translator.trans('address') }}
             </dt>
             <dd
-              v-if="locationContacts.street"
+              v-if="locationContacts?.street"
```
Comment (hwiem) [inline client/js/components/procedure/admin/DpAddOrganisationList.vue:34]: If `locationContacts` is not defined, wouldn't that mean that only the text "address" is displayed, but no address data? So maybe we could add a `v-if="locationContacts"` to the `dl` above, which should also make adding all the `?` below unnecessary?

File: client/js/components/procedure/admin/DpAddOrganisationList.vue [hunk 2]
```diff
@@ -179,8 +179,8 @@ export default {
 
     rowItems () {
       return Object.values(this.invitableToebItems).reduce((acc, item) => {
-        const locationContactId = item.relationships.locationContacts.data[0].id
-        const locationContact = this.getLocationContactById(locationContactId)
+        const locationContactId = item.relationships.locationContacts.data.length > 0 ? item.relationships.locationContacts.data[0].id : null
```
Comment (hwiem) [inline client/js/components/procedure/admin/DpAddOrganisationList.vue:182]: Is `locationContacts` always defined?

File: client/js/components/procedure/admin/DpAddOrganisationList.vue (hunk 1 above)
Comment (gruenbergerdemos) [reply client/js/components/procedure/admin/DpAddOrganisationList.vue]: No, if there is no adress data, the translation `Translator.trans('notspecified')` is shown. But changed it a bit here to make it more clear: [23f1f9b](https://github.com/demos-europe/demosplan-core/pull/3649/commits/23f1f9bbf351eaf5ada2a7c69d9a9ce530170ab7)

File: client/js/components/procedure/admin/DpAddOrganisationList.vue (hunk 2 above)
Comment (gruenbergerdemos) [reply client/js/components/procedure/admin/DpAddOrganisationList.vue]: I added an additional check to be safe: [2f9be6a](https://github.com/demos-europe/demosplan-core/pull/3649/commits/2f9be6a36bae17e98ba62533279f2cea6315f8e3)

File: client/js/components/procedure/admin/DpAddOrganisationList.vue (hunk 1 above)
Comment (gruenbergerdemos) [reply client/js/components/procedure/admin/DpAddOrganisationList.vue]: And here: [beba1ae](https://github.com/demos-europe/demosplan-core/pull/3649/commits/beba1ae0399c5261458267769c03810d3646a2a2) forgot the check in the method earlier

**Pull request merged**